WMF Framework/CacheFileWriterHelper.swift

```diff
@@ -88,6 +88,11 @@ final class CacheFileWriterHelper {
         replaceFileWithData(headerData, fileName: fileName, completion: completion)
     }
     
+    static func dataForFileName(_ fileName: String) -> Data? {
```

Comment:
Is this used anywhere? Perhaps I'm missing it.

Reply:
Nope! Good catch, I used it in a since-deleted test that I could never get to run with Nocilla. This should be gone now.